modules/tinymce/src/core/main/ts/api/html/DomParser.ts

```diff
@@ -225,7 +225,7 @@ const transferChildren = (parent: AstNode, nativeParent: Node, specialElements:
   const parentName = parent.name;
   // Exclude the special elements where the content is RCDATA as their content needs to be parsed instead of being left as plain text
   // See: https://html.spec.whatwg.org/multipage/parsing.html#parsing-html-fragments
-  const isSpecial = Obj.has(specialElements, parentName) && parentName !== 'title' && parentName !== 'textarea';
+  const isSpecial = parentName in specialElements && parentName !== 'title' && parentName !== 'textarea';
```

Comment:
Now to start with, I know this isn't quite the same but our `Obj.has` implementation is a reasonable amount slower than doing this (between 2-3x slower). This meant that we were wasting ~15-20ms for the 1000 para test so I figured the pros outweighed the cons here but am happy to revert this if we want the absolute safety.